pocket_coffea/utils/plot_utils.py

```diff
@@ -53,6 +137,17 @@
     'bb': (0.51, 0.91, 0.51),  # green
 }
 
+colors_tthbb = {
```

Comment:
All of this needs to be part of the configuration of the particular analysis run, not of the plotting script... to be done in a separate PR

Reply:
Sure, this will be moved to the configuration file in a following PR featuring also a refactorization of the plotting code.